medusa/server/web/home/post_process.py

```diff
@@ -49,14 +49,20 @@ def _decode(value):
             proc_dir = _decode(proc_dir)
             resource_name = _decode(nzbName)
 
-            result = process_tv.ProcessResult(proc_dir, process_method=process_method).process(
-                resource_name=resource_name, force=argToBool(force), is_priority=argToBool(is_priority),
-                delete_on=argToBool(delete_on), failed=argToBool(failed), proc_type=type,
+            result = app.post_processor_scheduler.action.run(
```

Comment:
Something doesn't add up for me here.
So, if you call the `action`'s `run` directly, you're not *actually* running this threaded.
You need a something like a queue, so the `action` would be something like the implementation for `ShowQueue`.

Reply:
Okay maybe not a full-on queue, but just some way to trigger the scheduler to run with other custom arguments, (maybe also without changing the scheduled auto post-processing run)

Reply:
Well if you want to queue pp actions you need a queue. But then you need to poll, or have a callback.

Reply:
So the only difference with just calling a method, is that now you can ask the scheduler if the method is running.

Reply:
>So the only difference with just calling a method, is that now you can ask the scheduler if the method is running.

@p0psicles 
Yes, I agree with that.
What @medariox and I are disagreeing about is, that it runs in a thread (= not blocking).